.travis.yml

```diff
@@ -6,9 +6,9 @@ language: python
 env:
   global:
     # can be "stable", or anything that can be used with git checkout
-    - VALIDATOR_VERSION="master"
+    - VALIDATOR_VERSION="9d8a6ffae5e0ee117b97bc8c978fe34ec3c13af4"
```

Comment:
this points to the 1.3.0 release of bids-validator, 7 days before we released mne-bids: https://github.com/bids-standard/bids-validator/commit/9d8a6ffae5e0ee117b97bc8c978fe34ec3c13af4